tests/src/test/java/org/eclipse/hono/tests/registry/TenantAmqpIT.java

```diff
@@ -87,17 +110,23 @@ public static void shutdown(final TestContext ctx) {
     /**
      * Verifies that a client can use the get operation to retrieve information for an existing
      * tenant.
-     * 
+     *
      * @param ctx The vert.x test context.
      */
     @Test
     public void testGetTenant(final TestContext ctx) {
 
-        tenantClient
-            .get(Constants.DEFAULT_TENANT)
-            .setHandler(ctx.asyncAssertSuccess(tenantObject -> {
-                ctx.assertEquals(Constants.DEFAULT_TENANT, tenantObject.getTenantId());
-            }));
+        final String tenantId = Constants.DEFAULT_TENANT; //helper.getRandomTenantId();
+        final TenantObject payload = new TenantObject()
+                .setTenantId(tenantId)
+                .setEnabled(true)
+                .setTrustAnchor(getRandomPublicKey(), new X500Principal("CN=ca, OU=Hono, O=Eclipse"));
```

Comment:
no need to  slow down test execution by creating a key here, simply omit the trust anchor ...

Reply:
removed